docs/web/release_notes.md

```diff
@@ -0,0 +1,3 @@
+# Release Notes
+
+<!--next-version-placeholder-->
```

Comment:
FYI this is where new release notes based on commit logs will start to be automatically populated upon release.